fe/fe-core/src/test/java/org/apache/doris/nereids/rules/rewrite/EliminateSortTest.java

```diff
@@ -26,7 +26,7 @@
 /**
  * column prune ut.
  */
-public class EliminateSortTest extends TestWithFeService implements MemoPatternMatchSupported {
+class EliminateSortTest extends TestWithFeService implements MemoPatternMatchSupported {
```

Comment:
better to add ut to cover "select * from (select xxx order by xxx) a left join (select xxx ) b on a.id =b.id limit n;" also.